<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtUi</class>
 <widget class="QWidget" name="QtUi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>475</width>
    <height>475</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>475</width>
    <height>475</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>475</width>
    <height>475</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>A89 - Denis916 - Modes</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>390</y>
     <width>491</width>
     <height>86</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="modesLayout">
    <item>
     <widget class="QPushButton" name="adminButton">
      <property name="text">
       <string>Admin Mode</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="userCSVButton">
      <property name="text">
       <string>User CSV Mode</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="userHTMLButton">
      <property name="text">
       <string>User HTML Mode</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
